programmers/같은-숫자는-싫어.java

```diff
@@ -0,0 +1,25 @@
+import java.util.*;
+
+public class Solution {
+    public int[] solution(int []arr) {
+
+        
+        ArrayDeque<Integer> deque = new ArrayDeque<Integer>();
+        
+        for (int i = 0; i < arr.length; i++) {
+            if (deque.peek() == null) {
+                deque.addLast(arr[i]);
+            }
+            else if (deque.peekLast() != (arr[i])) {
+                deque.addLast(arr[i]);
+            }
+        }
+
+        int[] answer = new int[deque.size()];
+        int size = deque.size();
```

Comment:
배열 만들 때도 `size`를 사용하는 것이 좋을 것 같습니다.

```suggestion
        int size = deque.size();
        int[] answer = new int[size];
```